services/settings/pkg/store/defaults/defaults.go

```diff
@@ -33,6 +33,13 @@ const (
 	// LanguageReadWriteName is the hardcoded setting name for the language read write permission
 	LanguageReadWriteName string = "language-readwrite"
 
+	// DisableEmailNotificationsPermissionID is the hardcoded setting UUID for the language read write permission
+	DisableEmailNotificationsPermissionID string = "ad5bb5e5-dc13-4cd3-9304-09a424564ea8"
+	// DisableEmailNotificationsPermissionName is the hardcoded setting name for the language read write permission
+	DisableEmailNotificationsPermissionName string = "disable-email-notifications"
```

Comment:
If I recall correctly there was some decision to go with permission names like `EmailNotifications.Disable` in the future (first one was `PublicLink.Write`). Don't know the details of the decision and how to form permission names then, but this caught my eye...